README.md

```diff
@@ -1,5 +1,7 @@
 Docker Compose
 ==============
+![Docker Compose](https://github.com/docker/compose/blob/master/docs/logo.png).
```

Comment:
In Swarm they used this; https://github.com/docker/swarm/commit/accf600c327eeff416fb23d4ca12ad12b6ea89c5